web-wallet/src/routes/(welcome)/+layout.svelte

```diff
@@ -13,5 +23,51 @@
     height="31"
   />
 </header>
+{#key retryKey}
+  {#await networkStore.connect().then(() => networkStore.init())}
+    <div class="welcome-layout__loading">
+      <Throbber />
+      <strong>CONNECTING TO THE NETWORK</strong>
+    </div>
+    <!-- eslint-disable-next-line no-unused-vars -->
+  {:then _}
+    <slot />
+  {:catch error}
+    <Banner
+      className="welcome-layout__error"
+      title="Error while trying to connect to the network"
+      variant="error"
+    >
+      <p>
+        The web wallet can't connect to the network.<br />
+        It can be a temporary problem, so you can <AppAnchor
+          href="/setup"
+          on:click={handleRetry}>retry</AppAnchor
+        >
+        in a few minutes.
```

Comment:
I'd go with:

The Web Wallet is unable to connect to the network. This may be a temporary issue. Please try again in a few minutes by clicking retry.

Reply:
Fine by me